O/OpenSpiel/build_tarballs.jl

```diff
@@ -38,17 +27,23 @@ mv project_acpc_server-0.1.0 open_spiel/open_spiel/games/universal_poker/acpc
 
 mkdir open_spiel/build
 cd open_spiel/build
-BUILD_WITH_PYTHON=OFF BUILD_WITH_JULIA=ON BUILD_WITH_HANABI=ON BUILD_WITH_ACPC=OFF cmake -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DJulia_PREFIX=$Julia_PREFIX -DCMAKE_PREFIX_PATH=$prefix/destdir/lib/cmake/JlCxx -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_FIND_ROOT_PATH=${prefix} ../open_spiel/
+export BUILD_WITH_PYTHON=OFF BUILD_WITH_JULIA=ON BUILD_WITH_HANABI=ON BUILD_WITH_ACPC=OFF
+cmake \
+    -DCMAKE_FIND_ROOT_PATH=${prefix} \
+    -DCMAKE_INSTALL_PREFIX=$prefix \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DJulia_PREFIX=${prefix} \
+    ../open_spiel/
 make -j${nproc}
 make install
 install_license ${WORKSPACE}/srcdir/open_spiel/LICENSE
 """
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
-platforms = [
-    Platform("x86_64", "linux"; libc="glibc", cxxstring_abi = "cxx11"),
-]
+include("../../L/libjulia/common.jl")
+platforms = libjulia_platforms(julia_version)
+platforms = expand_cxxstring_abis(platforms)
```

Comment:
This part of the change expands the number of supported platforms substantially; but many currently don't build, due to various issues.  I note that upstream now is at version 0.2.0, so some of them may be fixed there (but of course also new ones may have been introduced).

One is in the  upstream C++ code:
```C++
 /workspace/srcdir/open_spiel/open_spiel/utils/json.cc:66:64: error: no matching function for call to ‘min(long unsigned int, std::basic_string_view<char>::size_type)’
SPIEL_CHECK_EQ(error, str.substr(0, std::min(30ul, str.size())));
```
Patching this with a fix is easy enough, but I am not sure it's worth the effort -- is there any interest in supporting more platforms?